internal/httpclient/upload.go

```diff
@@ -0,0 +1,71 @@
+package httpclient
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"mime/multipart"
+	"net/http"
+	"os"
+	"path/filepath"
+)
+
+// constant variables for HTTP request
+const (
+	POST         = "POST"
+	UploadSuffix = "v1/storage/upload"
+)
+
+// UploadFile uploads file to different regions
+func UploadFile(regionURL, fileName, formType string) ([]byte, error) {
+	// 1. prepare file
+	file, err := os.Open(fileName)
+	if err != nil {
+		return nil, err
+	}
+	defer file.Close()
+
+	body := &bytes.Buffer{}
+	writer := multipart.NewWriter(body)
+	part, err := writer.CreateFormFile(formType, filepath.Base(file.Name()))
+	if err != nil {
+		return nil, err
+	}
+	io.Copy(part, file)
+	writer.Close()
+
+	// 2. create request
+	request, err := http.NewRequest(POST, regionURL, body)
```

Comment:
`regionURL` is kind of a misnomer here. The request clearly has to hit the app storage service and not the base region URL.

Reply:
I'll rename that to `url`